net/NSString+stringIsEmpry.m

```diff
@@ -0,0 +1,28 @@
+//
+//  NSString+stringIsEmpry.m
+//  net
+//
+//  Created by user on 2/19/16.
+//  Copyright © 2016 Admin. All rights reserved.
+//
+
+#import "NSString+stringIsEmpry.h"
+
+@implementation NSString (stringIsEmpty)
+
++ (BOOL) stringIsEmpty:(NSString *)string {
+    if ((NSNull *) string == [NSNull null])
```

Comment:
How it works? It's just comparing of two pointers. It's not  a calling overloaded function as in Cpp 
(like bool operator==(SomeType &))